Reto002 - Alejandro De La Riva

AlejandroDeLaRivaGarcia opened the pull request.

AlejandroDeLaRivaGarcia committed:
Paso1 CreaciondeCarpetas

AlejandroDeLaRivaGarcia committed:
Paso2 CreacionArchivosTexto

AlejandroDeLaRivaGarcia committed:
Paso 3 ModificacionTextoArchivos

AlejandroDeLaRivaGarcia committed:
Paso4 MoverArchivosalDestino

AlejandroDeLaRivaGarcia committed:
Paso5 ModificarArchivosDestino

AlejandroDeLaRivaGarcia committed:
Cambio en el nombre

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

Commit:
Clase Java CalculoNotaFinal y Variables

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Add files via upload

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Piedra Papel o Tijera

mmasias commented on entregas/delarivaAlejandro/src/PiedraPapelTijera.java:
```diff
@@ -0,0 +1,28 @@
+
+        int aleatorio = (int)(Math.random() * (MAXIMO - MINIMO) + MINIMO);
+
+        String resultado = (eleccionDeUsuario == aleatorio) ? "Empate" :
+                           (eleccionDeUsuario == 1 && aleatorio == 3)|| 
+                           (eleccionDeUsuario == 2 && aleatorio == 1)||
+                           (eleccionDeUsuario == 3 && aleatorio == 2) ? "Has ganado" :
+                           "Has perdido";
```
Lo mismo que le he dicho a @juanQuintanaIng: https://github.com/mmasias/24-25-PRG1/pull/276#discussion_r1792475925

AlejandroDeLaRivaGarcia committed:
Añado Juegos For

AlejandroDeLaRivaGarcia committed:
Caracol

AlejandroDeLaRivaGarcia committed:
Caracol Final

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

AlejandroDeLaRivaGarcia committed:
Viaje de Marco pendiente madre

AlejandroDeLaRivaGarcia committed:
Viaje de Marco Resuelto

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

AlejandroDeLaRivaGarcia committed:
Añado Reto Carrera de Camellos

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

mmasias committed:
Update README.md

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

mmasias committed:
Segundo intento

mmasias committed:
3rd

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Enlaces

AlejandroDeLaRivaGarcia committed:
Archivos examen

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Reto Adivinación

mmasias committed:
eP2

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Subo Archivos que me faltaban

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Ejercicio de temperatura de 7 dias
- Con métodos estáticos
- Adecuado reparto de responsabilidades

mmasias committed:
Refactoriza manejo de temperaturas a matrices

- Cambia variables individuales de temperatura por una matriz.
- Actualiza métodos calcularMinima, calcularMaxima y calcularMedia para usar la nueva matriz.
- Simplifica el código de solicitud de temperaturas con un bucle for.

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Pequeñas anécdotas acerca de una matriz

mmasias committed:
Cambio extra

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Mapa de matrices

JavierLianoRioz committed:
Update README.md

mmasias committed:
Merge pull request #689 from JavierLianoRioz/patch-1

Modificación del Readme

mmasias committed:
pyMundo y su solución

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main

AlejandroDeLaRivaGarcia committed:
Añado problemas realizados en clase y tutorías como repaso

AlejandroDeLaRivaGarcia committed:
Añado ejercicio Mundo realizado en  tutorías

mmasias committed:
Update README.md

mmasias committed:
Update README.md

mmasias committed:
fixLink

AlejandroDeLaRivaGarcia committed:
Merge branch 'mmasias:main' into main
